### PR TITLE
Bump containerd to d97a907f

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>

This bumps containerd to d97a907f7f781c0ab8340877d8e6b53cc7f1c2f6. This is specifically to pick up this fix https://github.com/containerd/containerd/pull/2634 removing boltdb which has a bug on Windows, and moving to bbolt, the maintained fork with the fix instead. 

This PR directly relates to https://github.com/moby/moby/pull/37843 where there's a large dependency chain involved here to get all related components moved to bbolt. @thaJeztah @dmcgowan PTAL.

Note - there isn't a tagged release of v1.2 yet containing this fix, hence using the latest commit ID on master as at the time of writing.

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -115,7 +115,7 @@ github.com/googleapis/gax-go v2.0.0
 google.golang.org/genproto 694d95ba50e67b2e363f3483057db5d4910c18f9
 
 # containerd
-github.com/containerd/containerd v1.2.0-beta.2
+github.com/containerd/containerd d97a907f7f781c0ab8340877d8e6b53cc7f1c2f6
```
Comment (thaJeztah) [inline vendor.conf]: Is a cherry-pick opened for the patch to get it into the 1.2 branch in containerd?

**Review by thaJeztah — APPROVED** [review]
SGTM

**Comment by thaJeztah** [comment]
Oops, looks like there's an issue with vendoring;

```
16:11:10 The result of vndr differs
16:11:10 
16:11:10 ?? vendor/github.com/containerd/continuity/context.go
16:11:10 ?? vendor/github.com/containerd/continuity/digests.go
16:11:10 ?? vendor/github.com/containerd/continuity/groups_unix.go
16:11:10 ?? vendor/github.com/containerd/continuity/hardlinks.go
16:11:10 ?? vendor/github.com/containerd/continuity/hardlinks_unix.go
16:11:10 ?? vendor/github.com/containerd/continuity/hardlinks_windows.go
16:11:10 ?? vendor/github.com/containerd/continuity/ioutils.go
16:11:10 ?? vendor/github.com/containerd/continuity/manifest.go
16:11:10 ?? vendor/github.com/containerd/continuity/proto/
16:11:10 ?? vendor/github.com/containerd/continuity/resource.go
16:11:10 ?? vendor/github.com/containerd/continuity/resource_unix.go
16:11:10 ?? vendor/github.com/containerd/continuity/resource_windows.go
16:11:10 
```

**Comment by jhowardmsft** [comment]
Yeah, looks like containerd is pulling in new bits of continuity. Did a revendor on containerd/continuity without bumping the commit. Hopefully should work this time.

**Comment by jhowardmsft** [comment]
Nope, have had to bump containerd/continuity as well - now at f44b615e492bdfb371aae2f76ec694d9da1db537. Third time lucky....

File: vendor.conf (hunk 1 above)
Comment (jhowardmsft) [reply vendor.conf]: @dmcgowan Not sure how you're cutting 1.2, or just refreshing from master on each release?

**Comment by jhowardmsft** [comment]
And also bumped go-runc to 5a6d9f37cfa36b15efba46dc7ea349fa9b7143c3 to match containerd/containerd

**Comment by jhowardmsft** [comment]
Vendor passes now 😺

File: vendor.conf (hunk 1 above)
Comment (thaJeztah) [reply vendor.conf]: Oh, nevermind, I see that the 1.2-beta's are still tagged from master, and that there's no 1.2 branch yet

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Full diff for containerd; https://github.com/containerd/containerd/compare/v1.2.0-beta.2...d97a907f7f781c0ab8340877d8e6b53cc7f1c2f6

**Comment by jhowardmsft** [comment]
@dmcgowan @crosbymichael You guys OK with this? I should get this in first before revendoring moby/buildkit which already has the PR merged to bump its version of containerd and move to bbolt. Thx.

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**